internal/commands/server/create.go

```diff
@@ -283,6 +284,8 @@ func (s *createCommand) InitCommand() {
 
 // ExecuteWithoutArguments implements commands.NoArgumentCommand
 func (s *createCommand) ExecuteWithoutArguments(exec commands.Executor) (output.Output, error) {
+	fmt.Printf("\033[31m Plan: %s | Cores: %d | Mem: %d\033[0m", s.params.Plan, s.params.CoreNumber, s.params.MemoryAmount)
```

Comment:
This now displays 0 cores and mem for simple plans, which might be confusing.

(Also this is visible for just a moment before `logline.SetMessage` is called for the first time. Could be helpful to have logging mode which would keep all messages in view instead of replacing the line.)

Reply:
This was just a log that I used for quick check, removed the whole thing already :)